hathor/transaction/genesis.py

```diff
@@ -12,71 +12,75 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from typing import TYPE_CHECKING, Optional
+from typing import TYPE_CHECKING
 
-from hathor.conf import HathorSettings
+from hathor.conf.settings import HathorSettings
 from hathor.transaction import BaseTransaction, Block, Transaction, TxOutput
 
 if TYPE_CHECKING:
     from hathor.transaction.storage import TransactionStorage  # noqa: F401
 
-settings = HathorSettings()
 
-BLOCK_GENESIS = Block(
-    hash=settings.GENESIS_BLOCK_HASH,
-    nonce=settings.GENESIS_BLOCK_NONCE,
-    timestamp=settings.GENESIS_TIMESTAMP,
-    weight=settings.MIN_BLOCK_WEIGHT,
-    outputs=[
-        TxOutput(settings.GENESIS_TOKENS, settings.GENESIS_OUTPUT_SCRIPT),
-    ],
-)
-
-TX_GENESIS1 = Transaction(
-    hash=settings.GENESIS_TX1_HASH,
-    nonce=settings.GENESIS_TX1_NONCE,
-    timestamp=settings.GENESIS_TIMESTAMP + 1,
-    weight=settings.MIN_TX_WEIGHT,
-)
-
-TX_GENESIS2 = Transaction(
-    hash=settings.GENESIS_TX2_HASH,
-    nonce=settings.GENESIS_TX2_NONCE,
-    timestamp=settings.GENESIS_TIMESTAMP + 2,
-    weight=settings.MIN_TX_WEIGHT,
-)
-
-GENESIS = [BLOCK_GENESIS, TX_GENESIS1, TX_GENESIS2]
-
-GENESIS_HASHES = [settings.GENESIS_BLOCK_HASH, settings.GENESIS_TX1_HASH, settings.GENESIS_TX2_HASH]
-
-
-def _get_genesis_hash() -> bytes:
+def get_genesis_block(settings: HathorSettings) -> Block:
```

Comment:
I feel that we should assert `block.hash == GENESIS_BLOCK_HASH`.

Reply:
Done in https://github.com/HathorNetwork/hathor-core/pull/794/commits/a86c1756e42ce554a47e0048bbc19dec4614c377